packages/thirdweb/src/wallets/in-app/web/lib/iframe-wallet.ts

```diff
@@ -316,7 +316,7 @@ export class IFrameWallet implements IWebWallet {
                 parsedTypedData.types as SignerProcedureTypes["signTypedDataV4"]["types"],
               message:
                 parsedTypedData.message as SignerProcedureTypes["signTypedDataV4"]["message"],
-              chainId: chainId || 1,
+              chainId: Number.parseInt(BigInt(chainId || 1).toString()),
```

Comment:
was this intended?

Reply:
yes because typescript failed on it - not sure how it didn't fail before